### ISSUE TITLE
For screenreader users, TinyMCE should have consistent label behavior with other input fields

### ISSUE THREAD

**Issue by jkies00** [tinymce/tinymce#9558]
## 📝 Provide a description of the new feature or improvement
When I navigate through this form, each form field has it label announced properly, as per accessibility standards, using `for` and `id` attributes. The black box at the bottom of these screenshots shows what is announced.
![image](https://github.com/tinymce/tinymce/assets/1789302/ef68ef9e-4f54-4e5f-bdac-fd66d540d393)

When I navigate to the TinyMCE instance, the label is never announced:
![image](https://github.com/tinymce/tinymce/assets/1789302/8ca32789-5983-4926-b56d-8c44b847b606)

I think the Rich Text Area instructions, shown in the screenshot above, should be preceded by announcing the label. In this case, "Charity Description".

### 🫶 What is the motivation?
Making a better experience for screenreader users.

### 🔗 What is the consequence of not having this feature?
Implementing consistent label behavior would provide some additional context to the screenreader user, which they are currently missing. Currently, they are aware they are in a Rich Text Area, but what is the Rich Text Area for?

### 🚦 How important would you rate the requested feature or improvement?
I think this is important, and should be the base functionality for the TinyMCE editor.

---

If you'd like to see this implemented sooner, add a 👍 reaction to this post.

👍

**Comment by TheSpyder** [comment]
This is generally a limitation of `contenteditable` editors, and iframe-based editors in particular (which TinyMCE is by default).

The best we've been able to offer is an option to make this text customisable.
https://www.tiny.cloud/docs/tinymce/latest/accessibility/#iframe_aria_text
